packages/domain/src/use-cases/profile/UpdateProfileDetails.ts

```diff
@@ -5,12 +5,17 @@ import {
 } from '../transactions/DelegableProtocolCallUseCase';
 import { IProtocolCallPresenter } from '../transactions/ProtocolCallUseCase';
 
+export type ProfileAttributeValue = Date | string | number | null;
+
+export type PartialAttributesUpdate = Record<string, ProfileAttributeValue>;
+
 export type ProfileDetails = {
-  name: string;
+  // attributes: PartialAttributesUpdate;
```

Comment:
Although not used in this PR, this was useful for me to reason around the update path.

This PR implementation is all about the read path. `Profile#attributes` is this object that contains all custom attributes.

In this file you can have a glimpse of what changes are going to happen in the update path to complement the read.

```ts
export type ProfileDetails = {
  attributes: PartialAttributesUpdate;
  name: string;
  bio: string | null;
}
```